romanesco/__init__.py

```diff
@@ -19,9 +19,13 @@
     'docker': executors.docker.run,
     'python': executors.python.run,
     'r': executors.r.run,
-    'workflow': executors.workflow.run
+    'workflow': executors.workflow.run,
+    'pyspark': executors.pyspark.run
```

Comment:
Interesting to name it `pyspark`. This leaves open a native `spark` mode which takes Scala code. Is that the idea here?